test/util/core/iterable-to-string-test.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var referee = require("referee");
+var iterableToString = require("../../../lib/sinon/util/core/iterable-to-string");
+var assert = referee.assert;
+
+describe("util/core/iterable-to-string", function () {
+    it("returns an String representation of Array objects", function () {
+        var arr = [1, "one", true, undefined, null];
+        var expected = "1,'one',true,undefined,null";
+
+        assert.equals(iterableToString(arr), expected);
+    });
+
+    it("returns an String representation of Map objects", function () {
+        if (typeof Map === "function") {
```

Comment:
Yes it would be better since then this test would not count as passed, it would just be ignored.
Well noticed, thanks!